src/rich-text/node-views/code-block.ts

```diff
@@ -26,7 +26,7 @@ export class CodeBlockView implements NodeView {
 
         this.dom.innerHTML = escapeHTML`
 <pre class="s-code-block"><code class="content-dom"></code></pre>
-<select class="ps-absolute bg-transparent h:bg-unset c-pointer t2 r4 fs-fine us-none js-lang-select"></select>
+<div class="s-select s-select__sm ps-absolute t6 r6"><select class="js-lang-select"></select></div>
```

Comment:
You probably don't want that `</select>` in there now :)

Reply:
Not sure I follow? `select` is [not a valid self-closing tag](https://html.spec.whatwg.org/multipage/form-elements.html#the-select-element:concept-element-tag-omission), so the end tag is necessary.

Reply:
Oh, I misread that whole line. I thought `<select>` was replaced with a `<div>` altogether for some reason. My bad. 🤦‍♂️

Reply:
Nah, you're good. This is example number 10,134 on "why building html in strings sucks". I have a micro JSX library that is _this close_ to being made public just so I can pull it in to use in this project...